### ISSUE TITLE
Unary statements fails in GET v2/entities?q= if attribute has ":"

### ISSUE THREAD

**Issue by LeonanCarvalho** [telefonicaid/fiware-orion#2452]
The character `:` isn't a [forbidden character](http://fiware-orion.readthedocs.io/en/stable/user/forbidden_characters/) but when it's used to a unary statements on simple query fails because 

> "The symbol `:`  can be used instead of `=`"

For example:

`GET http://localhost:1026/v2/entities?q=!device:location`

OR

`GET http://localhost:1026/v2/entities?q=device:location`

Both will return a empty array because of this rule.

OCB Info:

```
{
  "orion" : {
  "version" : "1.2.0-next",
  "uptime" : "9 d, 17 h, 27 m, 37 s",
  "git_hash" : "c1a224db08086751f05d471a8b52c0eaec15f3a5",
  "compile_time" : "Thu Aug 4 09:04:12 BRT 2016",
  "compiled_by" : "root"
}
}
```

**Comment by fgalan** [comment]
So you mean that 

```
GET http://localhost:1026/v2/entities?q=device:location
```

returns a "forbidden character error" ?

**Comment by LeonanCarvalho** [comment]
Not, If I have a attribute like "device:location" and I try to query entities with that attribute  it will return same as this request: 
GET http://localhost:1026/v2/entities?q=device=location

So, I can't use on `q`  attribute with `:` composing their name, maybe it should be a forbidden character.

**Comment by fgalan** [comment]
Probably `:` to `==` is extending to all the URL query string, not only to the cases which are mean to be operators. Based on that, it seems to be a bug, to be fixed in a next release.

(Note: you mention `=` as an equivalente for `:`, but according to documentation it should be `==`. Is there a typo in your report?)

Thanks!

**Comment by LeonanCarvalho** [comment]
Yes, there is a typo in my report, please consider `== `instead a single `=`.

**Comment by fgalan** [comment]
Fixed by PR https://github.com/telefonicaid/fiware-orion/pull/2830. Note that ' need to be used to avoid `:` to be interpreted as part of the filter. Thus, the @LeonanCarvalho 's examples would be as follow:

```
GET http://localhost:1026/v2/entities?q='device:location'
GET http://localhost:1026/v2/entities?q='!device:location'
```

**Comment by fgalan** [comment]
PR has been merged. @LeonanCarvalho do you want to try, in order to see if this is fixed? The fix will be in the next release (1.7) to be published by end of January, although if you don't want to wait you can also test directly with fiware/orion:latest docker (https://hub.docker.com/r/fiware/orion/).

**Comment by LeonanCarvalho** [comment]
Tested and worked as expected:

- http://localhost:1026/v2/entities?q=%27device:location%27 return all entities without device:location attr
- http://localhost:1026/v2/entities?q=!%27device:location%27 return all entities with device:locatio attr
